netpyne/analysis/lfp.py

```diff
@@ -536,4 +536,525 @@ def plotLFP(timeRange=None, electrodes=['avg', 'all'], plots=['timeSeries', 'PSD
     # show fig 
     if showFig: _showFigure()
 
+    return figs, outputData
+
+
+
+
+
+# -------------------------------------------------------------------------------------------------------------------
+## Plot LFP (time-resolved, power spectral density, time-frequency and 3D locations)
+# -------------------------------------------------------------------------------------------------------------------
+#@exception
```

Comment:
This needs to be changed to @exception (remove the #), otherwise any crash in this function will crash netpyne.

Reply:
(as discussed on slack -- keeping plotCSD() in csd.py!)